src/components/itemsCardsList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ItemCard from './itemCard';
+import { IItemData } from '../data/items';
+import './itemsCardsList.css';
+
+interface IItemsCardsListProps {
+  heightOffset: string;
+  baseImgUrl: string;
+  items: IItemData[];
+}
+
+function ItemsCardsList({ heightOffset, baseImgUrl, items }: IItemsCardsListProps) {
+  return (
+    <section
+      className="items-cards-list"
+      style={{ height: `calc(100vh - (${heightOffset}))` }}
+      data-testid="items-cards-list"
+    >
+      {items.map((item) => {
+        return (
+          <ItemCard
+            key={item.id}
+            title={item.title}
+            imgUrl={baseImgUrl + item.poster_path}
+            overview={item.overview}
+            popularity={item.popularity.toString()}
+            vote_average={item.vote_average.toFixed(1)}
```

Comment:
Please don't mix camelCase with snake_case in your codebase, it will be hard to maintain in the future

Reply:
Snake_case is used in API